apps/callscreen/js/calls_handler.js

```diff
@@ -295,12 +295,23 @@ var CallsHandler = (function callsHandler() {
       if (contact && contact.name) {
         CallScreen.incomingInfo.classList.add('additionalInfo');
         CallScreen.incomingNumber.textContent = contact.name;
-        CallScreen.incomingNumberAdditionalTelType.textContent =
-          Utils.getPhoneNumberAdditionalInfo(matchingTel);
+
+        var additionalL10n =
+          Utils.getLocalizedPhoneNumberAdditionalInfo(matchingTel);
+        if (additionalL10n.id) {
+          navigator.mozL10n.setAttributes(
+            CallScreen.incomingNumberAdditionalTelType,
+            additionalL10n.id,
+            additionalL10n.args);
+        } else {
+          CallScreen.incomingNumberAdditionalTelType.setAttribute(
```

Comment:
I'm curious, is there a specific reason why you're calling setAttribute() directly instead of using mozL10n.setAttributes()? I've used the latter form thorough the code for consistency. I think the above could also be shortened into:
```javascript
navigator.mozL10n.setAttributes(
  CallScreen.incomingNumberAdditionalTelType,
  additionalL10n.id || additionalL10n,
  additionalL10n.args || null);
```